### PR TITLE
[terra-menu] Sub menu height is restricting to the parent menu height

### PR DESCRIPTION
**Pull request by KV106606Viswanath**
### Summary
Previously, the submenu height was fixed to the main menu height when the menu items exceeded the viewport size. With this fix, the menu height now occupies the entire viewport height.

**What was changed:**
Removed the fixed height of the menu when menu items exceeded the viewport size.

**Why it was changed:**

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-10234

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-menu/src/_MenuContent.jsx [hunk 1]
```diff
@@ -187,7 +187,7 @@ class MenuContent extends React.Component {
     if (this.props.index > 0 && this.listNode) {
       const bufHeight = ((this.context.name || this.context.className) === 'orion-fusion-theme') || MenuUtils.isSafari() ? 20 : 10;
       const submenuHeight = this.listNode.clientHeight + this.listNode.parentNode.parentNode.parentNode.firstChild.clientHeight + bufHeight;
-      return submenuHeight > window.innerHeight ? this.props.fixedHeight : submenuHeight;
+      return submenuHeight;
```
Comment (sugan2416) [inline packages/terra-menu/src/_MenuContent.jsx:190]: what happens if its set to window height ?

```suggestion
      return submenuHeight > window.innerHeight ? window.innerHeight : submenuHeight;;
```

File: packages/terra-menu/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed menu height issue when it has more items than viewport height.
```
Comment (sugan2416) [inline packages/terra-menu/CHANGELOG.md:6]:
```suggestion
  * Fixes sub-menu height issue when it exceeds viewport size.
```

**Comment by sugan2416** [comment]
Add examples and tests for large submenu items and also the reverse case

File: packages/terra-menu/src/_MenuContent.jsx (hunk 1 above)
Comment (KV106606Viswanath) [reply packages/terra-menu/src/_MenuContent.jsx]: Updated

File: packages/terra-menu/CHANGELOG.md (hunk 2 above)
Comment (KV106606Viswanath) [reply packages/terra-menu/CHANGELOG.md]: Updated

**Comment by KV106606Viswanath** [comment]
> Add examples and tests for large submenu items and also the reverse case


> Add examples and tests for large submenu items and also the reverse case

Added

**Review by sugan2416 — APPROVED** [review]

**Review by saket2403 — APPROVED** [review]

**Pull request merged**